sdk/keyvault/azure-security-keyvault-certificates/samples/certificate_basic_operations.md

```diff
@@ -91,3 +91,6 @@ If the Azure Key Vault is soft delete-enabled and you want to permanently delete
 auto result = response.PollUntilDone(defaultWait);
 certificateClient.PurgeDeletedCertificate(certificateName);
 ```
+## Source
+
+[Source Code](https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-basic-operations)
```

Comment:
To increase visibility and access, maybe we should instead put this link in the beginning of the doc for folks to reference right away, rather than as a separate heading at the end of the doc (which folks might not read through to get to).

It could be linked in the first sentence of the readme, like follows:

[This sample] demonstrates how to: (https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-basic-operations)

Reply:
The quick and relevant links are in the top of each Package's README.

See KeyVault, for example: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-keys#azure-key-vault-key-client-library-for-c

Reply:
![image](https://user-images.githubusercontent.com/24213737/148992542-7aa3ff90-f54c-492b-8a58-85afaf629c6b.png)

Reply:
@vhvb1989 that's for keyvault-keys.

I don't see how we get to the samples README doc pages like this one `certificate_basic_operations.md` specifically for certificates.

Currently the flow is the following:
1) The user lands on this page:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-certificates/README.md
2) They scroll down to:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-certificates/README.md#next-steps
3) Then, they get to:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-certificates/samples/certificate_import_certificate.md
4) And now, they have to scroll all the way to the bottom, to get to the sample source.

I was recommending moving that link to the top of the sample-specific doc, so they can get to the sample source code right away, if they wanted to.